tools/hacky_postprocess_around_closure_limitations.py

```diff
@@ -0,0 +1,41 @@
+# This file further processes an output JS file to optimize down to even smaller size that Closure is unable to do
+import sys, re
+
+f = open(sys.argv[1], 'r').read()
+orig_size = len(f)
+
+f = f.strip()
+
+if '"use asm";' in f:
+	f = re.sub(r'\s*/\*\* @suppress {uselessCode} \*/\s*', '', f)
+	f = re.sub(r'\s*//\s*EMSCRIPTEN_START_FUNCS\s*', '', f)
```

Comment:
why not remove all comments?

Reply:
Only these comments are added after the Closure pass. Closure will already have removed all other comments from the file.